.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,68 @@
+name: '🐞 Bug report'
+description: Let us know about functionality that is not working as expected
+title: "Bug report: <short description>"
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting the issue you are facing.
+        Please complete the below form to ensure we have the necessary details to assist you.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Provide a short description of the issue you are facing
+    validations:
+      required: true
```

Comment:
>required: true [](http://example.com/codeflow?start=6&length=14)

Please make this optional (for situations where this is just a repeat of the title).